File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
  int n;
  do {
    n = get_int("Height: ");
  } 
  while (n >= 8 || n <= 1);
   //  int j = 1;
   for (int i = 0; i < n; i++){
       printf(" ");
     for (int j = 0; j < i ; j++){
         printf("#");
     }
  printf("\n");
   }

 // j++;
   
  return 0;
}

File: mario.py
from cs50 import get_int

i = get_int("Height: ")
if i < 1 and i > 8:
    i = get_int("Height:")

k = 1
h = i - 1
while k <= i and h >= 0:
    for j in range(i):
        print(" "*h + "#"*k)
        k = k + 1
        h = h - 1
